From 0f805c1a59417d947adc834821f9e407720029d5 Mon Sep 17 00:00:00 2001
From: Chun-wei Fan <fanchunwei@src.gnome.org>
Date: Tue, 21 Mar 2017 16:39:04 +0800
Subject: [PATCH] gettext-tools/gnulib-lib/xalloc.h: Fix function signature

This ensures we use _Noreturn across the board so that the compiler
won't complain about different prototype signatures, and expands to
support different compilers.
---
 gettext-tools/gnulib-lib/xalloc.h | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/gettext-tools/gnulib-lib/xalloc.h b/gettext-tools/gnulib-lib/xalloc.h
index f4a329e..0fcb44c 100644
--- a/gettext-tools/gnulib-lib/xalloc.h
+++ b/gettext-tools/gnulib-lib/xalloc.h
@@ -60,11 +60,7 @@ extern "C" {
    in charge of honoring the three previous items.  This is the
    function to call when one wants the program to die because of a
    memory allocation failure.  */
-extern void xalloc_die (void)
-#if (__GNUC__ > 2 || (__GNUC__ == 2 && __GNUC_MINOR__ >= 5)) && !__STRICT_ANSI__
-     __attribute__ ((__noreturn__))
-#endif
-     ;
+extern _Noreturn void xalloc_die (void);
 
 /* In the following macros, T must be an elementary or structure/union or
    typedef'ed type, or a pointer to such a type.  To apply one of the
-- 
2.10.2.windows.1

